src/pages/Auth.tsx

```diff
@@ -86,8 +83,9 @@ const Auth = () => {
                         success();
                     }
                 })
-                .catch(() => {
+                .catch(async (error) => {
                     trackEvent('exception');
+                    await failed(error.message);
```

Comment:
I think this is how some users got stuck on the loading page forever - so making sure we navigate away when there is a huge error.